python/tvm/topi/hexagon/utils.py

```diff
@@ -49,4 +59,8 @@ def get_layout_transform_fn(layout):
         return n11c_1024c_2d
     if layout == "n11c-1024c-1d":
         return n11c_1024c_1d
+    if layout == "nhwc-1024c-1d":
+        return nhwc_1024c_1d
+    if layout == "nc-1d":
+        return nc_1024_1d
```

Comment:
You should change the layout string "nc-1d" -> "nc_1024c_1d" since that's what the actual physical layout of the buffer seems like.